src/wallet/wallet.cpp

```diff
@@ -6713,15 +6713,23 @@ bool CWallet::ParameterInteraction(const CChainParams& params)
     if (mapArgs.count("-maxtxfee"))
     {
         CAmount nMaxFee = 0;
+        CAmount lowMaxTxFee = CalculateConventionalFee(LOW_LOGICAL_ACTIONS);
         if (!ParseMoney(mapArgs["-maxtxfee"], nMaxFee))
             return UIError(AmountErrMsg("maxtxfee", mapArgs["-maxtxfee"]));
         if (nMaxFee > HIGH_MAX_TX_FEE)
             UIWarning(_("-maxtxfee is set to a very high fee rate! Fee rates this large could be paid on a single transaction."));
         maxTxFee = nMaxFee;
-        if (maxTxFee < CalculateConventionalFee(LOW_LOGICAL_ACTIONS))
+        if (CFeeRate(maxTxFee, 1000) < ::minRelayTxFee)
         {
-            return UIError(strprintf(_("Invalid amount for -maxtxfee=<amount>: '%s' (must allow for at least %d logical actions at the conventional fee)"),
-                                       mapArgs["-maxtxfee"], LOW_LOGICAL_ACTIONS));
+            return UIError(strprintf(_("Invalid amount for -maxtxfee=<amount>: '%s' (must be at least the minimum relay fee rate of %s to prevent stuck transactions)"),
```

Comment:
```suggestion
            return UIError(strprintf(_("Invalid amount for -maxtxfee=<amount>: '%s' (must be at least the minimum relay fee of %s for a 1000-byte transaction, to prevent stuck transactions)"),
```
I honestly don't think this makes any sense at all. It is comparing a fee with a fee rate. If the transaction is more than 1000 bytes, it is useless.

Reply:
Both `maxTxFee` and `minRelayTxFee` are configurable, so this is providing a check that you aren't setting `-maxtxfee` to a provably unsatisfiable value. You are correct that even with this check, there will be a range of transaction sizes that could be unsatisfiable, but that range will not cover all possible transactions.